src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const div = document.getElementById('div');
+const logo = document.getElementById('logo');
+
+function handler() {
+  const promise1 = (resolve, reject) => {
+    logo.addEventListener('click', () => {
+      resolve('<div class="message">Promise was resolved!</div>')
+    });
+  };
+
+  return new Promise(promise1);
+}
+
+handler().then(function(result) {
+  div.innerHTML = result;
+});
```

Comment:
Use arrow function as a callback
```suggestion
handler().then(result => {
  div.innerHTML = result;
});
```